### PR TITLE
#FT-1(Design): home page

### PR DESCRIPTION
**Pull request by chrisostome01**
# PR Description

This Pull Request aims to add design for the landing page of the Medical core

# Description of what this task covered

- [x]  CSS and HTML only
- [x]  BEM methodology

# How this may be tested?

1. Clone the repo
4. Navigate into fuzzy-kmarket
3. Check out to the appropriate branch(ft/home-page-design)
4. Open index.html in your browser or you can use your live server extension

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width= , initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="./style/main.css">
+    <title>KMarketer</title>
+</head>
+<body>
+    <div class="content">
```
Comment (Jonath-z) [inline index.html:12]: `content` block name  is not referenced nowhere in the block, you can removed it or rename it

File: index.html [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        <div class="header header--bg-gray">
+            <div class="header__item ">
+                <div class="logo">
+                    <h1 class="">KMarketer</h1>
+                </div>
+                <ul class="navigation">
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">Home</a>
```
Comment (Jonath-z) [inline index.html:20]: `link` is good , however i can suggest you to reference it to its block like `navigation__link` then the modifier will be `navigation__link--disabled`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+                    </li>
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">
+                            <img src="./assets/svgs/cat.svg" alt="">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="header__content">
+                <h1 class="site-header site-header--large " >
```
Comment (Jonath-z) [inline index.html:44]: `site-header site-header--large` those class are global I guess, isn't there a way to avoid them. cause coming to meaning, this does not means something in the below code.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+                        dog the quick fox jumps over the 
+                        lazy dog
+                    </div>
+                </div>
+            </div>
+        </div>
+    
+        <div class="video">
+            <div class="video__headline">
+                <h1 class="site-header site-header--medium site-header--center" >
```
Comment (Jonath-z) [inline index.html:156]: like at this line `site-header site-header--medium site-header--center` are not meaningfull

**Review by Jonath-z — CHANGES_REQUESTED** [review]
Kindly applie those review

File: style/main.css [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+.site-header{
+    color:#252B42;
+    font-weight: 800;
+}
+.site-header--medium {
+    font-size: 30px;
+}
+.site-header--large {
+    font-size: 50px;
+}
```
Comment (Samiskina) [inline style/main.css:89]: these modifiers are all for one site-header, how about you create one that has all the modification

File: index.html [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+            <div class="main-video">
+                <div class="video__image">
+                    <img class="site-video" src="./assets/svgs/video.svg">
+                </div>
+            </div>
+            <div class="container">
+                <h1 class="site-header site-header--medium site-header--center" >
+                    What Clients Say
+                </h1>
+                <div class="site-content">
```
Comment (Samiskina) [inline index.html:175]: how about you name this depending on container, one would think they are very separated while it is in the container

**Review by Samantha-KY — COMMENTED** [review]
even though there are some nested blocks, you can still have a block that is named according to the first block

File: index.html (hunk 6 above)
Comment (sezeranoJchrisostome) [reply index.html:173]: Am going to hide this all comment because my pr is still in progress

**Comment by chrisostome01** [comment]
> even though there are some nested blocks, you can still have a block that is named according to the first block

Thank you! 
@Samiskina I think that would be a block element and I won't be able to re-use it elsewhere.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,444 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="./style/main.css">
+    <title>KMarketer</title>
+</head>
+<body>
+    <div class="content">
+        <div class="header header--bg-gray">
+            <div class="header__item">
+                <h1 class="logo">KMarketer</h1>
+                <ul class="navigation">
```
Comment (Jonath-z) [inline index.html]: try to reference the `navigation` to the header  as it's a direct element of heade.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,444 @@
...
+                <h1 class="site-header site-header--large " >
+                    Experts are here solve 
+                    <br/>
+                    your business problem.
+                </h1>
+                <p class="header__sub-content">
+                    We know how large objects will act, but things on a 
+                    <br/>small scale just do not act that way.
+                </p>
+                <div class="header__action">
```
Comment (Jonath-z) [inline index.html]: `header__action` does not have any meaning , may be `header__button-container` makes sens

File: index.html [hunk 9]
```diff
@@ -0,0 +1,444 @@
...
+                    </button>
+                    <button class="button button--secondary-outline button--rounded-full" >
+                        Learn More
+                    </button>
+                </div>
+            </div>
+        </div>
+    
+        <div class="container container--bg-white">
+            <h1 class="site-header site-header--medium site-header--center" >
```
Comment (Jonath-z) [inline index.html]: `site-header`, `site-header--medium`, `site-header--center` are global, and here in this block those naming don't have  any meaning . try to improve it

File: index.html [hunk 10]
```diff
@@ -0,0 +1,444 @@
...
+        </div>
+    
+        <div class="video">
+            <div class="video__headline">
+                <h1 class="site-header site-header--medium site-header--center" >
+                    We are providing best 
+                    <br>
+                    business service.
+                </h1>
+                <div class="site-content">
```
Comment (Jonath-z) [inline index.html]: `site-content` does not have any meaning here

File: index.html [hunk 11]
```diff
@@ -0,0 +1,444 @@
...
+                    Problems trying to resolve the conflict between <br>
+                    the two major realms of Classical physics: Newtonian mechanics 
+                </div>
+            </div>
+            <div class="main-video">
```
Comment (Jonath-z) [inline index.html:196]: this block has chidren but for any of its childreni can see it reference , improve it too

File: index.html [hunk 12]
```diff
@@ -0,0 +1,444 @@
...
+                <div class="video__image">
+                    <img class="site-video" src="./assets/svgs/video.svg">
```
Comment (Jonath-z) [inline index.html]: `site-video` does not mean anything here. no reference to the parent .

File: index.html [hunk 13]
```diff
@@ -0,0 +1,444 @@
...
+                </div>
+            </div>
+        </div>
+    
+        <div class="container">
+            <h1 class="site-header site-header--medium site-header--center" >
```
Comment (Jonath-z) [inline index.html]: to improve to this class does not have any meaning in  this  block

File: index.html [hunk 14]
```diff
@@ -0,0 +1,444 @@
...
+            <div class="site-content">
+                Problems trying to resolve the conflict between <br>
+                the two major realms of Classical physics: Newtonian mechanics 
+            </div>
+            <div class="client">
+                <div class="client__card">
+                    <div class="client__rate">
+                        <img src="./assets/svgs/rates.svg" alt="ratings">
+                    </div>
+                    <div class="client__description">
```
Comment (Jonath-z) [inline index.html]: `client-description` is good but no better i thing, `client__card-description` my be better for meaning

File: index.html [hunk 15]
```diff
@@ -0,0 +1,444 @@
...
+                        Slate helps you see how many 
+                        more days you need to work to 
+                        reach your financial goal.
+                    </div>
+                    <div class="client__profile">
```
Comment (Jonath-z) [inline index.html]: same thing here

File: index.html [hunk 16]
```diff
@@ -0,0 +1,444 @@
...
+                        <img class="client__image" src="./assets/img/Ellipse.png" alt="client image">
```
Comment (Jonath-z) [inline index.html]: here too and for each element in this `client__card`

File: index.html [hunk 17]
```diff
@@ -0,0 +1,444 @@
...
+                                Designer
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>    
+        
+        <div class="container container--bg-white">
+            <h1 class="site-header site-header--medium site-header--center" >
```
Comment (Jonath-z) [inline index.html]: those class don't mean anything in this block

File: index.html [hunk 18]
```diff
@@ -0,0 +1,444 @@
...
+                Meet Our Team
+            </h1>
+            <div class="site-content">
+                Problems trying to resolve the conflict between <br/>
+                the two major realms of Classical physics: Newtonian mechanics 
+            </div>
+            <div class="team">
+                <div class="team-card">
```
Comment (Jonath-z) [inline index.html]: `team-card` must be block__element like `team__card`

File: index.html [hunk 19]
```diff
@@ -0,0 +1,444 @@
...
+                    <img class="team-card__image" src="./assets/img/team1.png" alt="">
```
Comment (Jonath-z) [inline index.html]: you may improve this like `team__card-image`

File: index.html [hunk 20]
```diff
@@ -0,0 +1,444 @@
...
+                    <div class="team-card__name">
```
Comment (Jonath-z) [inline index.html]: here as well and on each element in this block

File: index.html [hunk 21]
```diff
@@ -0,0 +1,444 @@
...
+                        <img class="social__link" src="./assets/svgs/fbvector.svg" alt="facebook">
+                        <img class="social__link" src="./assets/svgs/instavesctor.svg" alt="instagram">
+                        <img class="social__link" src="./assets/svgs/twitterVector.svg" alt="twitter">
+                    </div>
+                </div>
+            </div>
+        </div>
+    
+        <div class="contact-us contact-us--spacing-midium ">
+            <div class="container container--spacing-small">
```
Comment (Jonath-z) [inline index.html]: improve the naming here , `container` in `contact-us`  by referencing this element to its block

File: index.html [hunk 22]
```diff
@@ -0,0 +1,444 @@
...
+                <div class="contact-us__title">
+                    Contact Us
+                </div>
+                <h1 class="site-header site-header--spacing-sm" >
```
Comment (Jonath-z) [inline index.html]: this naming does not have any meaning in this block

File: index.html [hunk 23]
```diff
@@ -0,0 +1,444 @@
...
+                    Make an Appointment
+                </h1>
+            </div>
+            <div class="container container--spacing-small">
```
Comment (Jonath-z) [inline index.html]: two `container` element in a block is not BEM

File: index.html [hunk 24]
```diff
@@ -0,0 +1,444 @@
...
+                    <textarea class="contact-us__field" name="" id="" cols="30" rows="10" placeholder="Message"></textarea>
+                    <button class="button button--primary button--rounded button--size-small" >
+                        Book Appointment
+                    </button>
+                </form>
+            </div>
+        </div>
+
+        <div class="footer footer--dark">
+            <div class="container">
```
Comment (Jonath-z) [inline index.html]: `container` element is not referenced no where , you can improve the naming here

File: index.html [hunk 25]
```diff
@@ -0,0 +1,444 @@
...
+                <div class="footer__nav">
+                    <h1 class="logo logo--white">KMarketer</h1>
```
Comment (Jonath-z) [inline index.html:380]: reference `logo` to `footer__nav`

File: index.html [hunk 26]
```diff
@@ -0,0 +1,444 @@
...
+                    <div class="social">
```
Comment (Jonath-z) [inline index.html:381]: reference `social` element to it `block`

File: index.html [hunk 27]
```diff
@@ -0,0 +1,444 @@
...
+                        <img class="social__link" src="./assets/svgs/youtube.svg" alt="facebook">
+                    </div>
+                </div>
+                <div class="footer__line"></div>
+                <div class="footer__links">
+                    <div class="footer__list">
+                        <div class="footer__title">
+                            Company Info
+                        </div>
+                        <ul class="navigation navigation--col navigation--right">
```
Comment (Jonath-z) [inline index.html]: reference `navigation` to its block `footer`

File: index.html [hunk 28]
```diff
@@ -0,0 +1,444 @@
...
+                            <li class="navigation__item">
+                                <a class="link link--disabled" href="#">Unlimited Support</a>
+                            </li>
+                        </ul>                 
+                    </div> 
+                    <div class="footer__list">
+                        <div class="footer__title">
+                            Resources
+                        </div>
+                        <ul class="navigation navigation--col navigation--right">
```
Comment (Jonath-z) [inline index.html]: same thing here

File: style/main.css [hunk 29]
```diff
@@ -0,0 +1,383 @@
...
+.button--secondary-outline{
+    border: 1px solid #00A0C1;
+    color: #00A0C1;
+}
+.button--rounded-right {
+    border-top-right-radius: 6px ;
+    border-bottom-right-radius: 6px ;
+}
+.button--size-small {
```
Comment (Jonath-z) [inline style/main.css]: no need to specify that the size is small , `button--small` make more sens

**Review by Jonath-z — CHANGES_REQUESTED** [review]
kindly apply requested changes

File: index.html (hunk 7 above)
Comment (chrisostome01) [reply index.html]: if I make that navigation the element there, I won't be able to re-use it

File: index.html (hunk 8 above)
Comment (sezeranoJchrisostome) [reply index.html:51]: Ooh that makes, I will rename it

File: index.html (hunk 9 above)
Comment (chrisostome01) [reply index.html]: This is a block, and am re-using it in every section if I make it an element I don't think that would be Okay because  I will be copying all styles which are the same over and over. I think I can change the name maybe

File: index.html (hunk 23 above)
Comment (sezeranoJchrisostome) [reply index.html:309]: This is a container and it's modifier

File: index.html [hunk 30]
```diff
@@ -0,0 +1,444 @@
...
+        <div class="container container--bg-white">
+            <h1 class="site-header site-header--medium site-header--center" >
+                WHY CHOOSE US
+            </h1>
+            <div class="site-content">
+                Problems trying to resolve the conflict between <br> 
+                the two major realms of Classical physics: Newtonian mechanics 
+            </div>
+            <div class="service">
+                <div class="card">
```
Comment (Samantha-KY) [inline index.html]: why not service card?

File: index.html [hunk 31]
```diff
@@ -0,0 +1,444 @@
...
+                        dog the quick fox jumps over the 
+                        lazy dog
+                    </div>
+                </div>
+            </div>
+        </div>
+    
+        <div class="video">
+            <div class="video__headline">
+                <h1 class="site-header site-header--medium site-header--center" >
```
Comment (Samantha-KY) [inline index.html]: isn't this an element of the video or video__headlines?

**Review by Samantha-KY — COMMENTED** [review]
the naming is quite hard to follow as the contents in blocks are mostly becoming blocks too

File: index.html (hunk 31 above)
Comment (chrisostome01) [reply index.html]: No, it's a section header

**Review by Samantha-KY — APPROVED** [review]
good!!

File: index.html (hunk 25 above)
Comment (chrisostome01) [reply index.html:380]: The logo is a block, actually

File: index.html (hunk 27 above)
Comment (chrisostome01) [reply index.html]: Navigation is A block and this was the second time I used it, I changed the name if you considered navigation as the header navigation it just lists of element which are ordered in different way

File: index.html (hunk 9 above)
Comment (chrisostome01) [reply index.html]: I think site-header, has its own meaning and can stand alone, I changed the name to section-header so that it more descriptive

File: index.html (hunk 26 above)
Comment (chrisostome01) [reply index.html:381]: Social, is a block and I have re-used it, I don't know how I can define another block yet I have one which does the same thing.

File: index.html (hunk 10 above)
Comment (chrisostome01) [reply index.html]: I renamed it to, section-content

File: index.html [hunk 32]
```diff
@@ -164,12 +195,14 @@ <h1 class="section-header section-header--medium section-header--center" >
...
-                    <img class="main-video__video" src="./assets/svgs/video.svg">
+                    <div class="main__video-play main__video-play--rounded-full">
+                        <img src="./assets/svgs/playVector.svg" alt="Play icon">
+                    </div>
                 </div>
             </div>
         </div>
     
-        <div class="container">
+        <div class="container container--spacing-larg">
```
Comment (Jonath-z) [inline index.html]: do you mean `container--spacing-large` ? If yes then add the missing `e`

File: style/main.css [hunk 33]
```diff
@@ -22,6 +22,11 @@
     align-items: center;
     padding: 27px 85px;
 }
+.header__icons {
+    display: none;
+    list-style: none;
+    gap: 1rem;
```
Comment (Jonath-z) [inline style/main.css]: you don't nedd gap here cause your block is displayed `none`

File: style/responsiveness.css [hunk 34]
```diff
@@ -19,4 +15,93 @@
...
+
+    .section-header {
+        text-align: center;
+        font-size: 40px;
+        margin-top: 2rem;
+    }
+    .header__button-container {
+        flex-direction: column;
+    }
+    .section-header {
```
Comment (Jonath-z) [inline style/responsiveness.css]: `section-header` is twice repeated in this code , on line 25 and 33 , the line 33 overwrite the margin setted on line 25, try to fixe this

File: style/responsiveness.css [hunk 35]
```diff
@@ -19,4 +15,93 @@
...
+        background-size: cover;
+    }
+    .container {
+        margin-top: 1rem;
+        padding: 0px;
+    }
+    .container--spacing-larg {
+        margin-top:3rem;
+    }
+    .header__content {
```
Comment (Jonath-z) [inline style/responsiveness.css]: restructure this CSS file bro. where `contact-us` is styled , is not where you can find the header's elements. improve this as well

File: style/responsiveness.css [hunk 36]
```diff
@@ -19,4 +15,93 @@
...
+    }
+    .header__navigation--mobile {
+        display: flex;
+        flex-direction: column;
+        font-size: 30px;
+    }
+    .button--mobile-size-full {
+        width: 100%;
+    }
+    .contact-us__form {
```
Comment (Jonath-z) [inline style/responsiveness.css]: move this where contact's block is syled.

**Review by Jonath-z — CHANGES_REQUESTED** [review]
we're almost there, just small details to improve. kindly apply them

**Review by Jonath-z — APPROVED** [review]
LGTM

**Review by karemarsy — APPROVED** [review]

File: index.html [hunk 37]
```diff
@@ -0,0 +1,494 @@
...
+                    business service.
+                </h1>
+                <div class="section-content">
+                    Problems trying to resolve the conflict between <br>
+                    the two major realms of Classical physics: Newtonian mechanics 
+                </div>
+            </div>
+            <div class="main-video">
+                <div class="main-video__image">
+                    <div class="main__video-play main__video-play--rounded-full">
```
Comment (karemarsy) [inline index.html]: `main__video-play` is positioned outside the Block `(main)`.
and also 
`main__video-play--rounded-full` is positioned outside the Block `(main)`.

File: index.html [hunk 38]
```diff
@@ -0,0 +1,494 @@
...
+                        <img src="./assets/svgs/rates.svg" alt="ratings">
+                    </div>
+                    <div class="client__card-description">
+                        Slate helps you see how many 
+                        more days you need to work to 
+                        reach your financial goal.
+                    </div>
+                    <div class="client__card-profile">
+                        <img class="client__card-image" src="./assets/img/Ellipse.png" alt="client image">
+                        <div class="cleint__card-information">
```
Comment (karemarsy) [inline index.html]: `cleint__card-information ` is positioned outside the Block `(cleint)`. or it's a mistake you wanted to write  `client__card-information`

File: index.html [hunk 39]
```diff
@@ -0,0 +1,494 @@
...
+                        <img src="./assets/svgs/rates.svg" alt="ratings">
+                    </div>
+                    <div class="client__card-description">
+                        Slate helps you see how many 
+                        more days you need to work to 
+                        reach your financial goal.
+                    </div>
+                    <div class="client__card-profile">
+                        <img class="client__card-image" src="./assets/img/Ellipse1.png" alt="client image">
+                        <div class="cleint__card-information">
```
Comment (karemarsy) [inline index.html]: `cleint__card-information` is positioned outside the Block `(cleint)`. or it's a mistake you wanted to write `client__card-information`

File: index.html [hunk 40]
```diff
@@ -0,0 +1,494 @@
...
+                        <img src="./assets/svgs/rates.svg" alt="ratings">
+                    </div>
+                    <div class="client__card-description">
+                        Slate helps you see how many 
+                        more days you need to work to 
+                        reach your financial goal.
+                    </div>
+                    <div class="client__card-profile">
+                        <img class="client__card-image" src="./assets/img/Ellipse2.png" alt="client image">
+                        <div class="cleint__card-information">
```
Comment (karemarsy) [inline index.html]: `cleint__card-information` is positioned outside the Block `(cleint)`. or it's a mistake you wanted to write `client__card-information`

File: index.html [hunk 41]
```diff
@@ -0,0 +1,494 @@
...
+                            <div class="cleint__card-username">
+                                Regina Miles
+                            </div>
+                            <div class="cleint__card-title">
```
Comment (karemarsy) [inline index.html]: `cleint__card-title` is positioned outside the Block `(cleint).`

**Review by karemarsy — COMMENTED** [review]
hello @sezeranoJchrisostome it seems there are some resolved comments

**Review by karemarsy — COMMENTED** [review]
hello, @sezeranoJchrisostome  did you resolve my comments?

File: index.html (hunk 38 above)
Comment (chrisostome01) [reply index.html]: It's mistake,

Comment (chrisostome01) [reply index.html]: It's a mistake,

**Review by karemarsy — APPROVED** [review]
